CMakeLists.txt

```diff
@@ -183,7 +183,7 @@ option(USE_LRINT "Use lrint/lrintf for converting floating point values to integ
 option(USE_CURL "Use CURL library" ON)
 option(BUILD_RELEASE "Build for release" ON)
 option(BUILD_INSTALLER "Build installer" OFF)
-option(BUILD_TESTS "Build tests" ON)
+option(BUILD_TESTS "Build tests" OFF)
```

Comment:
Ah I missed this above. Any reason to do so? Just to avoid build breakages for normal users?

Reply:
yes, and because tests are more important for us than the normal users